.github/workflows/preview.yaml

```diff
@@ -0,0 +1,21 @@
+on: 
+  pull_request:
```

Comment:
should we add the action to destroy the preview when the PR is closed?

Reply:
Not that important from the article's perspective, so I skipped that. Do you want me to add it?

Reply:
every PR to this repo would leave preview ens running for 15 days no?

Reply:
Oh I forgot that would lead to this. Updated the PR!